src/options.ts

```diff
@@ -0,0 +1,11 @@
+// MerkleTree building options
+export type MerkleTreeOptions = Partial<{
+  // Enable or disable sorted leaves. Sorting is strongly recommended for multiproofs.
```

Comment:
This format is needed for the comment to be considered documentation:

```suggestion
  /** Enable or disable sorted leaves. Sorting is strongly recommended for multiproofs. */
```